apps/complaints/acceptance/features/confirmation.js

```diff
@@ -11,14 +11,16 @@ Before((
   I.visitPage(confirmationPage, steps);
 });
 
-Scenario('The correct fields elements are on the page', function *(
+Scenario('The correct page is served', function *(
   I,
   confirmationPage
 ) {
-  yield I.setSessionData(steps.name, {
-    'email-address': 'test@test.com'
-  });
+  yield I.amOnPage('/' + confirmationPage.url);
```

Comment:
This is implicitly checked on every page in the `visitPage` method called in the `Before`